index.bs

```diff
@@ -265,11 +265,18 @@ although they may be harder to find.
 Simpler features are easier for user agents to implement and test,
 more likely to be interoperable,
 and easier for authors to understand.
+If you expect authors will use your feature often,
+it's especially important to make it easy to use.
```

Comment:
I think we should recommend that each API should try to make its common cases easy to use regardless of how frequently the API is used in general. It's much easier for an API designer to know which use cases of their API are common, than to know how frequently an API is used compared to other APIs. 
That said, of course if API designers actually _know_ that their feature will be used often in general they should strive even more to make common cases easy, but like I said, that's often hard to gauge.